packages/react/src/components/Checkbox/index.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { boolean, select, text, withKnobs } from "@storybook/addon-knobs";
+
+import Checkbox from ".";
+
+storiesOf("Form controls/Checkbox", module)
+  .addDecorator(withKnobs)
```

Comment:
I wonder if we could move this into the main stories definition to avoid doing it every time 🤔

Reply:
👍

Reply:
Done